zcash_primitives/src/transaction/builder.rs

```diff
@@ -779,6 +782,68 @@ impl<P: consensus::Parameters, U: sapling::builder::ProverProgress> Builder<'_,
             .map_err(FeeError::FeeRule)
     }
 
+    #[cfg(zcash_unstable = "nu7")]
+    pub fn get_fee_zsa<FR: FeeRule + ZSAFeeRule>(
```

Comment:
Add doc + comment that mutual parts with Orchard can be extracted after the removal of the nu7 flag.
(explain the parameters in the doc, especially `is_asset_newly_created`).